Monika After Story/game/script-topics.rpy

```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
         m 3hua "Because I know that you're a good person despite how things are between us."
 
     elif mas_isMoniLove(higher=True):
-        m 1esc "I don't."
-        pause 3.0
-        m 1hub "Just kidding!"
-        m 1eub "You mean {i}everything{/i} to me, silly!"
-        m 1esd "But if you really want to know, [player]..."
-        # TODO: wink / tongue stick out here woul dbe perfect
-        m 1hfu "That's a secret!"
-
+        if renpy.random.randint(1,2) == 1:
+            if store.mas_anni.pastSixMonths():
+                m 1eka "Why do I even love you?"
+                m 2eka "[player]...{w=1} there are so many reasons why... Where do I even begin?"
```

Comment:
1 second waits are kinda long. Suggest using them very rarely. 0.5 is a better choice
Also no space as not a new sentence here, no space
```suggestion
                m 2eka "[player]...{w=0.5}there are so many reasons why... Where do I even begin?"
```